### ISSUE TITLE
confusion on dataset names

### ISSUE THREAD

**Issue by sainaasani** [vacancy/NSCL-PyTorch-Release#6]
Hello,
Can anyone explain me what is the difference between 'scenes-raw.json' and 'scenes.json' files? The scenes.json files with direct link have the exact same format data as in clevr dataset scenes files.

**Comment by vacancy** [comment]
The `scenes-raw.json` refers to the original `scenes.json` that could be downloaded from the CLEVR website. We actually do NOT use the file.

The `scenes.json` contains image information with detected bounding boxes, which is the file that we actually use. Copied from the README file:


> Next, you need to add object detection results for scenes. Here, we use the tools provided by [ns-vqa](https://github.com/kexinyi/ns-vqa).
In short, a pre-trained Mask-RCNN is used to detect all objects. We provide the json files with detected object bounding boxes at [clevr/train/scenes.json](http://nscl.csail.mit.edu/data/code-data/clevr/train/scenes.json.zip) and [clevr/val/scenes.json](http://nscl.csail.mit.edu/data/code-data/clevr/val/scenes.json.zip).
